### PR TITLE
Add to README: setup to run pytest unit tests

### PR DESCRIPTION
**Pull request by dchaley**
Hi Van Valen Lab 👋🏻 

I'm a software developer trying to learn bioinformatics. I've heard great things about DeepCell and have been playing around with it. I ran some benchmarking and found that post-processing took quite a while for very large images (~600 MB). I see that "it is known" that the `h_maxima` algorithm is slow. I'm curious to profile why by adding some log entries + profiling the `deep_watershed` function (which afaict is the core of Mesmer post-processing?)

In any case, here's what I needed to do to get the unit tests passing. Does this look reasonable to include in the readme? Happy to update as needed! (Is there some other way y'all do this?)

### CODE REVIEW

**Comment by dchaley** [comment]
Hi @willgraf @ngreenwald @msschwartz21 @rossbar – just wanted to put this on your radar! Not sure how to add y'all as PR reviewers (I think I forgot to when I opened it in the first place).

**Comment by ngreenwald** [comment]
Hey @dchaley, 
Thanks for looking into this. Yes, the `deep_watershed` function is the core workhorse for post-processing, and is also where the slowdown comes from. From what I remember, in addition to `h_maxima`, the `fill_holes` function was also quite slow. 

Looks good to me, I'll wait for Ross to weigh in.

File: README.md [hunk 1]
```diff
@@ -7,3 +7,27 @@
...
+#     .venv\Scripts\activate
+
+python3.10 -m pip install --editable .
+
+# Optionally: (to have ipython in the venv)
+python3.10 -m pip install ipython
+# Re-activate the venv if you installed ipython.
+
+# Run unit tests
+pytest
```
Comment (rossbar) [inline README.md:32]: Another nit, but I typically use `--pyargs` to ensure I'm getting the installed version. In principle there shouldn't be a difference with editable installs, but if users end up doing non-editable installs (e.g. they forget the -e flag), then this will still work

```suggestion
pytest --pyargs deecell_toolbox
```

**Review by rossbar — APPROVED** [review]
This generally seems fine, though I'm personally -0.5 on providing prescriptions for setting up developer environments since there are so many other possible workflows. That being said, if we are going to document this, this is certainly the workflow I prefer (i.e. relying on builtins).

I left some suggestions, mostly advocating for brevity - feel free to pick-and-choose what you agree with then we'll merge. Thanks @dchaley

**Comment by dchaley** [comment]
Thanks @rossbar @ngreenwald ! The pointers are much appreciated 😁   I incorporated all the suggestions plus adjusted the wording re: preferred environment. LMK what you think!

By the way I have some great news actually. I reimplemented scikit's `grayreconstruct` using Cython-optimized fast-hybrid-reconstruct. https://github.com/dchaley/deepcell-imaging/pull/27
Here's the [benchmarking notebook](https://github.com/dchaley/deepcell-imaging/blob/dbc56213b42f442dc1d428d7ae1caedbd9bb311e/benchmarking/h_maxima/h_maxima_benchmark.ipynb) I used. I borrowed y'all's Mesmer sample data 🙌🏻 

Tldr, I'm getting:
~6x improvement over scikit h_maxima for 512x512 images.
~18x speedup for at least one 10k x 14k image.

I'm in process of getting more test data to make sure it actually works. On that note, if you have suggestions for validating & benchmarking further– please let me know. I'd be happy to package it up into a PR to `deepcell-toolbox`, or, package a separate Python library which builds the extension to add as a dependency, or, I may add it to scikit itself. (But that would likely take longer than getting it to you.)

Anyhow, it's been really fun working with DeepCell & learning about the field. Thanks for the `fill_holes` pointer! I'll take a look at that next 👀

**Comment by rossbar** [comment]
CI failure unrelated, thanks @dchaley !
